_data/chains/eip155-7210.json

```diff
@@ -8,10 +8,11 @@
     "symbol": "NIBI",
     "decimals": 18
   },
+  "status": "deprecated",
   "infoURL": "https://nibiru.fi",
   "shortName": "nibiru-testnet-1",
-  "chainId": 7210,
-  "networkId": 7210,
+  "chainId": 6910,
+  "networkId": 6910,
   "icon": "nibiru",
```

Comment:
Note that in other deprecated Chain ID files, it uses the deprecated "chainId" and "networkId", not the new values.